examples/webview_example/package-lock.json

```diff
@@ -70,9 +71,17 @@
       "integrity": "sha1-DELU+xcWDVqa8eSEus4cZpIsGyE="
     },
     "ansi-escapes": {
-      "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-2.0.0.tgz",
-      "integrity": "sha1-W65SvkJIeN2Xg+iRDj/Cki6DyBs="
+      "version": "3.1.0",
```

Comment:
How about the other changes in the file. Are we changing anything other than this version number?

Reply:
No other changes